presto-main/src/main/java/com/facebook/presto/execution/QueryStats.java

```diff
@@ -373,12 +373,18 @@ public static QueryStats create(
                 PlanFragment plan = stageInfo.getPlan().get();
                 for (OperatorStats operatorStats : stageExecutionStats.getOperatorSummaries()) {
                     // NOTE: we need to literally check each operator type to tell if the source is from table input or shuffled input. A stage can have input from both types of source.
+                    // NOTE: also check for operator names from presto native execution.
                     String operatorType = operatorStats.getOperatorType();
-                    if (operatorType.equals(ExchangeOperator.class.getSimpleName()) || operatorType.equals(MergeOperator.class.getSimpleName())) {
+                    if (operatorType.equals(ExchangeOperator.class.getSimpleName()) ||
+                            operatorType.equals(MergeOperator.class.getSimpleName()) ||
+                            operatorType.equals("Exchange") ||
```

Comment:
Not sure if it would be good to declare some string constants for Native operator names.
If that's the case, then where?

Reply:
Can we pass the second check if it is for native execution? I think we have a utility for this?